tests/test_specfile.py

```diff
@@ -371,11 +377,11 @@ def test_write_lang_files_base(self):
         """
         write_lang_files base test
         """
-        self.specfile.locales = ["pkg1", "pkg2"]
+        self.specfile.locales = ["/pkg1", "/pkg2"]
         self.specfile.write_lang_files()
         expect = ["\n%files locales",
-                  " -f pkg1.lang",
-                  " -f pkg2.lang",
+                  " -f /pkg1.lang",
+                  " -f /pkg2.lang",
```

Comment:
So, the -f option is special, since it directs rpm to read contents from a file, and the file usually lives in the build directory. Leaving off the leading slash is correct here.

Reply:
Fixing the test.

Reply:
Great, that looks better, thanks.